lib/core/src/lib/comments/comments.component.html

```diff
@@ -1,7 +1,7 @@
 <div class="adf-comments-container">
-    <h1 id="comment-header" class="adf-comments-header">
+    <h3 id="comment-header" class="adf-comments-header" role="heading">
```

Comment:
Same here for h3 tag

Reply:
Btw h tag shouldn't have `heading` role as the tag implies it itself

Reply:
h1 tag changed to h2 as per the discussion as h1 is already present at the top level and also heading role removed.